### ISSUE TITLE
Signature docs error in API docs for `MorphAnalysis.__contains__`

### ISSUE THREAD

**Issue by schorfma** [explosion/spaCy#13406]
Hello spaCy team,

I encountered another small error in the API docs.

---

For the method `MorphAnalysis.__contains__` in the API docs, the description for the `return` value actually describes the method argument, which is missing.

```mdx
| Name        | Description                                   |
| ----------- | --------------------------------------------- |
| **RETURNS** | A feature/value pair in the analysis. ~~str~~ |
```

### Suggested Changes

```diff
- | Name        | Description                                   |
- | ----------- | --------------------------------------------- |
- | **RETURNS** | A feature/value pair in the analysis. ~~str~~ |

+ | Name         | Description                                                           |
+ | ------------ | --------------------------------------------------------------------- |
+ | `feature`    | A feature/value pair. ~~str~~                                         |
+ | **RETURNS**  | Whether the feature/value pair is contained in the analysis. ~~bool~~ |
```

## Which page or section is this issue related to?

* Docs: [API Docs / `Morphology` / `MorphAnalysis.__contains__`](https://spacy.io/api/morphology#morphanalysis-contains)
* Code: [`website/docs/api/morphology.mdx` / Lines 138-152](https://github.com/explosion/spaCy/blob/3dc445d/website/docs/api/morphology.mdx?plain=1#L138-L152)

**Comment by svlandeg** [comment]
Thanks for the report! Your fix looks correct and I contributed it as https://github.com/explosion/spaCy/pull/13433 (feel free to open a PR directly for these kind of fixes!)

**Comment by schorfma** [comment]
> Thanks for the report! Your fix looks correct and I contributed it as #13433 (feel free to open a PR directly for these kind of fixes!)

Thank you.

Then, I will prepare a combined pull request for #13405 and additional typos I stumbled upon (For the latter I did not open issues).

Greetings
Martin
